config.js

```diff
@@ -1,5 +1,15 @@
 const config = {
-  port: 27121
+  port: 27121,
+  extensions: {
+    "C": "c",
+    "C++": "cpp",
+    "Python": "py"
+  },
+  compilers: {
+    "C": "gcc",
+    "C++": "g++",
+    "Python": "python"
```

Comment:
Would `python3` be better? In my Linux PC, `python` points to python2 and to use Python 3, I had to use `python3`. My guess is that if Python 3 is installed, then `python3` will definitely point to Python 3. I'm not sure since I have little experience with Python.

To reduce confusion, in the settings and other user-facing screens, we can say, "Compilation flags for Python 3" specifically.

Reply:
Also, can you add a small checking mechanism to see if python3 is installed, and if not, show an error to the user like "`python3` does not point to a valid Python installation" or something like that?

Reply:
> Right now we don't do anything if `gcc` or `g++` is not installed. I guess, we can show an info box in the corner if any of the compilation fails due to compiler/interpreter not being installed.

Yeah, I can do that. Also I wrote a snippet for it but according to what you said I think it should be ignored for now .